docs/deployment.md

```diff
@@ -159,6 +160,182 @@ Deploy Probot as a Serverless Function to [ZEIT Now](http://zeit.co). After [cre
+    <img src="https://platform.sh/images/deploy/lg-blue.svg" alt="Deploy on Platform.sh" width="180px" />
+</a>
+</p>
+
+The link will create a free trial account on Platform.sh, create a new project to host the app, and then deploy the [Probot template repository](https://github.com/platformsh-templates/probot) on that project.
+
+After that, there are only a few steps left to set up and register the app:
+
+1. Install the Platform.sh CLI
+
+  Setting relevant environment variables and retrieving information about the Probot project will be much easier once you install the [Platform.sh CLI](https://docs.platform.sh/development/cli.html). Run the following command:
+
+  ```
+  curl -sS https://platform.sh/cli/installer | php
+  ```
+
+  You can find the system requirements and more information in the [installation instructions on GitHub](https://github.com/platformsh/platformsh-cli/blob/master/README.md#installation).
+
+1. Follow the remaining steps in the README
+
+  The template is designed to set the `APP_ID`, `PRIVATE_KEY`, and `WEBHOOK_SECRET` environment variables automatically for you during the registration process, by installing and using the [Platform.sh CLI](https://docs.platform.sh/development/cli.html) within your application container.
+
+  Platform.sh allows you to branch your production application and deploy an exact copy of that application on staging and development environments that themselves can be registered with GitHub, becoming fully testable apps in their own right.
+
+  Follow the remaining steps in the [README](https://github.com/platformsh-templates/probot/blob/master/README.md) to finish setting up the template.
+
+1. Migrate your code
+
+    After you have completed the above steps, you can branch your local repository and move your code onto that branch. You can revert the `NODE_ENV` variable to go through registration again for that environment
+
+    ```
+    echo 'export NODE_ENV="development"' > .environment
+    ```
+
+    and then repeat the steps above to register test the app on that development environment. When you're satisfied that your app is working as expected, merge it into the production application.
+
+#### Deploying an existing project
+
+You can branch the template above, add your application's code to it, and merge to fully migrate your app to Platform.sh. Alternatively, you can manually configure your Probot app to deploy on Platform.sh with only a few changes to your repository.
+
+1. [Create a free trial account](https://accounts.platform.sh/platform/trial/general/setup)
+
+1. Create a new project
+
+  When you create a new account, the setup wizard will direct you to create a new project. Name it, and select a region for the project to be served from.
+
+1. Install the [Platform.sh CLI]([Platform.sh CLI](https://docs.platform.sh/development/cli.html))
```

Comment:
This renders as `1.` again. I think you need to indent the paragraph above by 3 spaces? Same is true for the list items below

Reply:
Thanks for the catch, updated